platforms/android/com/bugfender/sdk/cordova/BugfenderPlugin.java

```diff
@@ -23,176 +21,180 @@
 
 public class BugfenderPlugin extends CordovaPlugin {
 
-		private CallbackContext callback = null;
```

Comment:
It is not clear what has changed here, since the file has been reformatted too. Please make only the necessary changes.

Reply:
Sorry. I fixed it. Now the diff contains only the relevant changes.